Library/Homebrew/cask/quarantine.rb

```diff
@@ -172,5 +173,17 @@ def self.propagate(from: nil, to: nil)
 
       raise CaskQuarantinePropagationError.new(to, quarantiner.stderr)
     end
+
+    def self.copy_xattrs(from, to)
+      odebug "Copying xattrs from #{from} to #{to}"
+
+      system_command!(swift,
+                      args: [
+                        *swift_target_args,
+                        COPY_XATTRS_SCRIPT,
+                        from,
+                        to,
+                      ])
```

Comment:
```suggestion
      system_command!(
        swift,
        args: [
          *swift_target_args,
          COPY_XATTRS_SCRIPT,
          from,
          to,
      ])
```

IMO is easier to follow but feel free to decline this, very optional